[BUG] templates/csi/daemonset.yaml: unable to parse YAML: error converting YAML to JSON: yaml

**What is your environment(Kubernetes version, Fluid version, etc.)**

**Describe the bug**

```
helm lint fluid
==> Linting fluid
[INFO] Chart.yaml: icon is recommended
[ERROR] templates/csi/daemonset.yaml: unable to parse YAML: error converting YAML to JSON: yaml: line 74: mapping values are not allowed in this context

Error: 1 chart(s) linted, 1 chart(s) failed
```

**What you expect to happen:**

**How to reproduce it**

**Additional Information**

--- a/charts/fluid/fluid/CHANGELOG.md
+++ b/charts/fluid/fluid/CHANGELOG.md
@@ -41,3 +41,4 @@
 
 * Add mountPropagation for registrar
 * Add syncRetryDuration
+* Add auto fuse recovery

--- a/charts/fluid/fluid/Chart.yaml
+++ b/charts/fluid/fluid/Chart.yaml
@@ -18,7 +18,7 @@ version: 0.7.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 0.7.0-f41c05c
+appVersion: 0.7.0-380b056
 home: https://github.com/fluid-cloudnative/fluid
 keywords:
   - category:data

--- a/charts/fluid/fluid/templates/csi/daemonset.yaml
+++ b/charts/fluid/fluid/templates/csi/daemonset.yaml
@@ -75,9 +75,9 @@ spec:
                 fieldRef:
                   fieldPath: spec.nodeName
             - name: NODE_IP
-                valueFrom:
-                  fieldRef:
-                    fieldPath: status.hostIP
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.hostIP
             {{- if .Values.runtime.mountRoot }}
             - name: MOUNT_ROOT
               value: {{ .Values.runtime.mountRoot | quote }}

--- a/charts/fluid/fluid/values.yaml
+++ b/charts/fluid/fluid/values.yaml
@@ -6,7 +6,7 @@ workdir: /tmp
 
 dataset:
   controller:
-    image: fluidcloudnative/dataset-controller:v0.7.0-f41c05c
+    image: fluidcloudnative/dataset-controller:v0.7.0-380b056
 
 csi:
   recoverFusePeriod: -1
@@ -15,7 +15,7 @@ csi:
   registrar:
     image: registry.aliyuncs.com/acs/csi-node-driver-registrar:v1.2.0
   plugins:
-    image: fluidcloudnative/fluid-csi:v0.7.0-f41c05c
+    image: fluidcloudnative/fluid-csi:v0.7.0-380b056
   kubelet:
     rootDir: /var/lib/kubelet
 
@@ -28,9 +28,9 @@ runtime:
     portRange: 20000-26000
     enabled: true
     init:
-      image: fluidcloudnative/init-users:v0.7.0-f41c05c
+      image: fluidcloudnative/init-users:v0.7.0-380b056
     controller:
-      image: fluidcloudnative/alluxioruntime-controller:v0.7.0-f41c05c
+      image: fluidcloudnative/alluxioruntime-controller:v0.7.0-380b056
     runtime:
       image: registry.aliyuncs.com/alluxio/alluxio:release-2.7.2-SNAPSHOT-3714f2b
     fuse:
@@ -44,19 +44,19 @@ runtime:
     fuse:
       image: registry.cn-shanghai.aliyuncs.com/jindofs/jindo-fuse:3.7.3
     controller:
-      image: fluidcloudnative/jindoruntime-controller:v0.7.0-f41c05c
+      image: fluidcloudnative/jindoruntime-controller:v0.7.0-380b056
     init:
       portCheck:
         enabled: false
-      image: fluidcloudnative/init-users:v0.7.0-f41c05c
+      image: fluidcloudnative/init-users:v0.7.0-380b056
   goosefs:
     runtimeWorkers: 3
     portRange: 26000-32000
     enabled: false
     init:
-      image: fluidcloudnative/init-users:v0.7.0-f41c05c
+      image: fluidcloudnative/init-users:v0.7.0-380b056
     controller:
-      image: fluidcloudnative/goosefsruntime-controller:v0.7.0-f41c05c
+      image: fluidcloudnative/goosefsruntime-controller:v0.7.0-380b056
     runtime:
       image: ccr.ccs.tencentyun.com/qcloud/goosefs:v1.1.0-kona_jdk11 
     fuse:
@@ -64,12 +64,12 @@ runtime:
   juicefs:
     enabled: false
     controller:
-      image: fluidcloudnative/juicefsruntime-controller:v0.7.0-f41c05c
+      image: fluidcloudnative/juicefsruntime-controller:v0.7.0-380b056
     fuse:
       image: juicedata/juicefs-csi-driver:v0.11.0
 
 webhook:
   enabled: true
-  image: fluidcloudnative/fluid-webhook:v0.7.0-f41c05c
+  image: fluidcloudnative/fluid-webhook:v0.7.0-380b056
   replicas: 1
 

